src/Parser/BalanceDruid/Modules/Items/EmeraldDreamcatcher.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Module from 'Parser/Core/Module';
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class EmeraldDreamcatcher extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  isEDBuff(event) {
+    const spellId = event.ability.guid;
+    return spellId === SPELLS.ED_BUFF.id;
+  }
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasHead(ITEMS.EMERALD_DREAMCATCHER.id);
+  }
+
+  dreamcatcherBuffDropped = 0;
+
+  on_toPlayer_removebuff(event) {
+    if (this.isEDBuff(event)) {
+      this.dreamcatcherBuffDropped++;
+    }
+  }
+
+  suggestions(when) {
+    const buffDropsPerMinute = Math.round((((this.dreamcatcherBuffDropped) / (this.owner.fightDuration / 1000)) * 60)*10) / 10;
+
+    when(buffDropsPerMinute).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span> You dropped the <SpellLink id={SPELLS.ED_BUFF.id} /> buff {this.dreamcatcherBuffDropped} times. Try to maintain it up at all times, you can consult the guide on ED usage <a href='http://goo.gl/mH8NVj' target='_blank' rel='noopener noreferrer'>here</a> </span>)
```

Comment:
I don't think `maintain it up` is proper English, how about `keep it up`? I think it would also read easier if the comma was actually a dot and the last part a new sentence. I also feel that last sentence is missing a motivator to consult the guide, so how about `For more information consult the <a href='http://goo.gl/mH8NVj' target='_blank' rel='noopener noreferrer'>guide on ED usage</a>.`?